tests/smoke/test_deeprec_model.py

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
We will probably change this file in the future, we use the smoke tests for testing notebook and making sure that the metrics are ok. We are developing in parallel an example for the other algos. Probably it will be faster if we have a quick chat

Reply:
OK.  Done with smoke test.  Will add test cases to integration part if necessary after this PR (to avoid conflicts in parallel development)